### ISSUE TITLE
Install page

### ISSUE THREAD

**Issue by SarahBaghdadi** [userfrosting/web#12]
Do we want an "install" page outside of the documentation, on the main UF site? The page could review the big picture of installation, then either send people to the documentation, or re-write the whole install process making use of pictures, colors, and typography.

**Comment by alexweissman** [comment]
Yeah!  What if we had some kind of friendly diagram, that showed the overall process of developing and producing a web application with UserFrosting.  Something like:

1. Set up development environment
2. Clone UF repo & run installation tool
3. Make sprinkles
4. Set up production environment
5. Deploy
6. Maintain
